build.xml

```diff
@@ -3,16 +3,17 @@
 <project name="Erfurt" default="build">
 
     <target name="build-unit" depends="prepare,lint,phpunit,phpdoc,phpcb" />
-    <target name="build-integration" depends="prepare,lint,phpunit-integration,phpdoc,phpcb,phpmd-ci,phpcpd,phpcs-ci,pdepend" />
-    <target name="build-full" depends="prepare,lint,phploc,pdepend,phpmd-ci,phpcs-ci,phpcpd,phpdoc,phpunit-integration,phpcb" />
+    <target name="build-integration" depends="prepare,lint,phpunit-integration,phpdoc,phpcb,phpcpd,phpcs-ci,pdepend" />
+    <target name="build-full" depends="prepare,lint,phploc,pdepend,phpcs-ci,phpcpd,phpdoc,phpunit-integration,phpcb" />
```

Comment:
Why did you remove phpmd?
It is also available with composer: https://packagist.org/packages/phpmd/phpmd
https://phpmd.org/